docs/getting-started/rulesets.md

```diff
@@ -80,11 +80,9 @@ Specifying the format is optional, so you can completely ignore this if all the
 rules:
   api-servers:
     description: "OpenAPI `servers` must be present and non-empty array."
-    recommended: true
     formats: ["oas3"]
-    given: "$"
+    given: "$.servers"
```

Comment:
To mitigate the risk, I'd suggest reverting these for now.
We can revisit them in a separate PR.
What do you think?
As I described in Phil's PR, field with given vs given with no field they are not equivalent in certain circumstances.

Reply:
@P0lip ACK. I'll revert the `given` related changes.